src/main/java/seedu/address/logic/commands/AddTagsCommand.java

```diff
@@ -0,0 +1,120 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.commands.EditCommand.createEditedPatient;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PATIENTS;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.patient.EditPatientDescriptor;
+import seedu.address.model.patient.Patient;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Adds one or more tags to the specified patient.
+ * Tags can only be a single word, with no space in between.
+ * Repeated tags in command will be added as a single tag.
+ * If the patient already has the tag, it will not be added.
+ */
+public class AddTagsCommand extends Command {
+
+    public static final String COMMAND_WORD = "addtags";
```

Comment:
```suggestion
    public static final String COMMAND_WORD = "addt";
```

Might be better to standardise to "addt" since Adding Patients is "addp" and Adding Important Dates is "adde"